schemas/compiler_pass_v1.json

```diff
@@ -1002,36 +999,23 @@
         "op_link_errors"
       ]
     },
-    "routing_config": {
+
+    "routingmethod" :{
       "type": "object",
-      "description": "A configuration for the routing procedure.",
+      "description": "A method used during circuit mapping.",
       "properties": {
-        "depth_limit": {
-          "type": "integer",
-          "minimum": 0,
-          "description": "The look ahead limit for SWAP picking."
-        },
-        "distrib_limit": {
-          "type": "integer",
-          "minimum": 0,
-          "description": "The look ahead limit for Distributed CX gate checking."
-        },
-        "interactions_limit": {
-          "type": "integer",
-          "description": "The number of interactions considered in Distributed CX gate checking."
+        "name": {
+          "type": "string",
+          "description": "String identifying method and whether it can be serialized."
```

Comment:
OK. Still it probably doesn't make much sense in the schema description of "name". I'd just say "Name of method"?

Reply:
I've changed the name

Reply:
Sorry, still a bit confused here. It now reads:
```
    "routingmethod" :{
      "type": "object",
      "description": "A method used during circuit mapping.",
      "properties": {
        "name_of_method": {
          "type": "string",
          "description": "String identifying method and whether it can be serialized."
        },
      },
    }
```
But I dsn't see how the `name_of_method` identifies whether it can be serialized; it is just a name like `"LexiRouteRoutingMethod"`.

Reply:
I think I've misunderstood what you wanted changed here.
Would changing the description to "Name of method" be fine?

Reply:
Yes, or even:
```
    "routingmethod" :{
      "type": "object",
      "description": "A method used during circuit mapping.",
      "properties": {
        "name": {
          "type": "string",
          "description": "Name of method."
        },
      },
    }
```
(Calling it `name_of_method` rather than just `name` seems redundant.)

Reply:
I've updated it